posixlib/src/main/scala/scala/scalanative/posix/pthread.scala

```diff
@@ -224,17 +216,15 @@ object pthread {
 
   def pthread_testcancel(): Unit = extern
 
-  // Types
-  type routine = CFuncPtr0[Unit]
+  // symbolic constants
```

Comment:
Yes!! and Double Yes!!   I think that you/w have arrived at a technically correct and useful wording.  As I touch files, I will try to propagate that wording instead of the useful-at-the-time but not quite correct "// Macros".